### ISSUE TITLE
32-bit output not working for gcc 6.1

### ISSUE THREAD

**Issue by travisdowns** [mattgodbolt/gcc-explorer#132]
Based on this [earlier issue](https://github.com/mattgodbolt/gcc-explorer/issues/41) it seems that 32-bit output is supported. However, when I try to get 32-bit output for gcc 6.1, I get:

```
/opt/gcc-6.1.0/bin/../lib/gcc/x86_64-linux-gnu/6.1.0/../../../../x86_64-linux-gnu/bin/ld: cannot find -lstdc++
/opt/gcc-6.1.0/bin/../lib/gcc/x86_64-linux-gnu/6.1.0/../../../../x86_64-linux-gnu/bin/ld: skipping incompatible /usr/lib/x86_64-linux-gnu/libm.so when searching for -lm
/opt/gcc-6.1.0/bin/../lib/gcc/x86_64-linux-gnu/6.1.0/../../../../x86_64-linux-gnu/bin/ld: skipping incompatible /usr/lib/x86_64-linux-gnu/libm.a when searching for -lm
/opt/gcc-6.1.0/bin/../lib/gcc/x86_64-linux-gnu/6.1.0/../../../../x86_64-linux-gnu/bin/ld: cannot find -lgcc_s
/opt/gcc-6.1.0/bin/../lib/gcc/x86_64-linux-gnu/6.1.0/../../../../x86_64-linux-gnu/bin/ld: skipping incompatible /opt/gcc-6.1.0/bin/../lib/gcc/x86_64-linux-gnu/6.1.0/libgcc.a when searching for -lgcc
```

This seems to indicate the 32-bit libs are not installed. This works on gcc 4.9 though.

Here's an example of a [failing compile](http://godbolt.org/#compilers:!((compiler:g6,options:'-O3+-m32+-march%3Di386',source:'class+Foo%0A%7B%0A++int+x%3B%0Apublic:%0A++void+f()%3B%0A++%0A%7D%3B%0A%0Avoid+Foo::f()+%7B%0A++x%2B%2B%3B%0A%7D%0A')),filterAsm:(binary:!t,commentOnly:!t,directives:!t,intel:!t,labels:!t),version:3).

**Comment by travisdowns** [comment]
It fails also on 4.9.2, 5.2 and 5.3.

**Comment by mattgodbolt** [comment]
Looks like I'm missing the 32-bit libraries. If you unselect the "binary"
mode it works ok: https://godbolt.org/g/C18TXo

**Comment by mattgodbolt** [comment]
As of now I've fixed 5.2, 5.3, 6.1 and 6.2. I'l hoping to rebuild 4.9.x compilers with similar support too.

**Comment by mattgodbolt** [comment]
Fixed for 4.9.4 too. Closing; as time ticks on I'll refresh more compilers, but 4.9.4 and above should all work now.
